lib/screenshot.js

```diff
@@ -0,0 +1,137 @@
+var https = require("https"),
```

Comment:
Missing spaces inside the parens. I know this is copy/pasted from the existing file, but I just noticed it.